packages/default/scss/calendar/_layout.scss

```diff
@@ -338,6 +338,7 @@
             .k-calendar-header {
                 margin-left: -$calendar-content-padding-x;
                 margin-right: -$calendar-content-padding-x;
+                padding: ($calendar-header-padding-y * 2) ($calendar-header-padding-x * 4);
```

Comment:
why is the variable called `$calendar-header-padding-` if we need to multiply it to get the values in calendar header?

Reply:
The idea is to use the `$calendar-header-padding-` variable (without multiplying it) for all other Calendar Headers and only multiply it for the Infinite Calendar Header.
Should I introduce a new variables instead, for example `$calendar-infinite-header-padding-`?